ngraph/core/reference/include/ngraph/runtime/reference/fake_quantize.hpp

```diff
@@ -194,7 +194,7 @@ namespace ngraph
                 }
 
             } // namespace fake_quantize_details
-            namespace v0
+            inline namespace v0
```

Comment:
Why do we need to have inline namespace here? Just to save backward compatibility?

Reply:
this whole namespace will be removed after cleanup in plugins, which was temporary switched to `v0` (function template generic call resolution problem).  `inline` should allow to drop explicitly namespace prefix in plugins.